scapy/contrib/gtp.py

```diff
@@ -531,7 +531,10 @@ class IE_GSNAddress(IE_Base):
     name = "GSN Address"
     fields_desc = [ByteEnumField("ietype", 133, IEType),
                    ShortField("length", 4),
-                   IPField("address", RandIP())]
+                   ConditionalField(IPField("ipv4_address", RandIP()),
```

Comment:
Do you think it's cleaner to have the IPv4 and IPv6 fields named separately, or both under `address` by using a `MultipleTypeField` ?

Reply:
A single name with `MultipleTypeField` would be probably nicer indeed, I just do not see any such field in the GTP parser and I wanted to be consistent with the existing implementation. I can change this if you prefer, just let me know please.